eschalot.c

```diff
@@ -105,7 +105,8 @@ strnlen(const char *s, size_t ml)
 #define RSA_KEYS_BITLEN	1024			/* RSA key length to use */
 #define SIZE_OF_E	4			/* Limit public exponent to 4 bytes */
 #define RSA_E_START	0xFFFFFFu + 2		/* Min e */
-#define RSA_E_LIMIT	0x7FFFFFFFu		/* Max e */
+#define RSA_E_LIMIT	0xFFFFFFFFu		/* Max e */
```

Comment:
Why is it relevant in this commit?

Reply:
I got it there by mistake due to a wrongly resolved merge. The following commit reverses that change.